src/Http/Client/Adapter/Stream.php

```diff
@@ -293,7 +293,7 @@ protected function _send(Request $request)
      */
     protected function _open($url)
     {
-        set_error_handler([$this, '_connectionErrorHandler']);
+        set_error_handler([$this, 'connectionErrorHandler']);
```

Comment:
benefit of real methods is stack traces in case of errors in tools like newrelic is smoother :)